packages/react-app/src/components/L2Bridge.jsx

```diff
@@ -59,7 +59,7 @@ export default function L2ArbitrumBridge({ address, userSigner }) {
     setProviders();
   }, [rollup, activeConfig.L1, activeConfig.L2]);
 
-  const contracts = useContractLoader(userSigner, { externalContracts: L1BridgeMetadata, hardhatContracts: {} });
+  const contracts = useContractLoader(userSigner, { externalContracts: L1BridgeMetadata, truffleContracts: {} });
```

Comment:
Looks like this whole file is still a WIP from the scaffold devs, but can we rename this to follow suit with the others?
```suggestion
  const contracts = useContractLoader(userSigner, { externalContracts: L1BridgeMetadata, deployedContracts: {} });
```